src/foundry/client-esm/applications/_module.d.mts

```diff
@@ -1,6 +1,6 @@
 // In Foundry itself this file contains re-exports of these other modules.
 // Therefore it has a runtime effect and uses `.mjs` instead of `.d.mts`.
-// While `.mts` could work, to avoid `import/no-unresolved from erroring `.mjs` is used.
+// While `.mts` could work, to avoid `import/no-unresolved` from erroring `.mjs` is used.
```

Comment:
Oops. Made this mistake once and then copy and pasted it everywhere. Thanks for the fix.